easybuild/easyconfigs/h/HiGHS/HiGHS-1.7.0-gfbf-2023b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'HiGHS'
+version = '1.7.0'
+
+homepage = 'https://ergo-code.github.io/HiGHS'
+description = """Open source serial and parallel solvers for large-scale sparse linear programming (LP),
+mixed-integer programming (MIP), and quadratic programming (QP) models."""
+
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+source_urls = ['https://github.com/ERGO-Code/HiGHS/archive/']
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}]
+checksums = ['d10175ad66e7f113ac5dc00c9d6650a620663a6884fbf2942d6eb7a3d854604f']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('SciPy-bundle', '2023.11'),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'download_dep_fail': True,
+    'use_pip': True,
+    'sanity_pip_check': True,
+    'installopts': '',
+}
+
+exts_list = [
+    ('highspy', version, {
+        'source_urls': ['https://github.com/ERGO-Code/HiGHS/archive/'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['d10175ad66e7f113ac5dc00c9d6650a620663a6884fbf2942d6eb7a3d854604f'],
+    }),
+]
+
```

Comment:
The sanity_check_paths part is missing from here. Should contain both the part from the previous HiGHS-GCC based file and a dirs check for 'lib/python%(pyshortver)s/site-packages'